package com.tsukimiai.hoshi.conversation.mapper;

import org.apache.ibatis.annotations.Mapper;

import com.baomidou.mybatisplus.core.mapper.BaseMapper;
import com.tsukimiai.hoshi.conversation.entity.UserMemory;

@Mapper
public interface UserMemoryMapper extends BaseMapper<UserMemory> {
}
